test/itemlist_formaction.cpp

```diff
@@ -53,14 +53,13 @@ TEST_CASE("process_op()", "[itemlist_formaction]") {
 			std::ifstream browserFileStream (browserfile.getPath());
 			openedItemsCount = 0;
 			if (browserFileStream.is_open()) {
-				for (int i = 0; i < maxItemsToOpen; i ++) {
-					if ( std::getline (browserFileStream,line) ) {
-						INFO("Each URL should be present exactly once. Erase urls after first match to fail if an item opens twice.")
-						REQUIRE(url_set.count(test_url + std::to_string(i)) == 1);
-						url_set.erase(url_set.find(test_url + std::to_string(i)));
-						openedItemsCount += 1;
-					}
+				while ( std::getline (browserFileStream,line) ) {
+					INFO("Each URL should be present exactly once. Erase urls after first match to fail if an item opens twice.")
+					REQUIRE(url_set.count(line) == 1);
+					url_set.erase(url_set.find(line));
+					openedItemsCount += 1;
 				}
+				REQUIRE(url_set.size() == itemCount - maxItemsToOpen);
```

Comment:
I don't think it makes sense to check this since it's the same as the next `REQUIRE` where you check how many items were opened. (Same goes for the similar lines in the rest of the test.)